DataManagementSystem/DB/FileCatalogComponents/WithFkAndPs/DirectoryClosure.py

```diff
@@ -522,33 +522,6 @@ def _setDirectoryParameter( self, path, pname, pvalue ):
     else:
       return DirectoryTreeBase._setDirectoryParameter( self, path, pname, pvalue )
 
-
-
-  def setDirectoryGroup( self, path, gname ):
```

Comment:
Please leave this file untouched, I will need these methods (with an underscore in front...)